build/crypto.m4

```diff
@@ -39,38 +39,28 @@ AC_DEFUN([TS_CHECK_CRYPTO], [
 
   dnl add checks for other varieties of ssl here
 ])
-dnl
-
-AC_DEFUN([TS_CHECK_CRYPTO_EC_KEYS], [
-  _eckeys_saved_LIBS=$LIBS
-
-  TS_ADDTO(LIBS, [$OPENSSL_LIBS])
-  AC_CHECK_HEADERS(openssl/ec.h)
-  AC_CHECK_FUNCS(EC_KEY_new_by_curve_name, [enable_tls_eckey=yes], [enable_tls_eckey=no])
-  LIBS=$_eckeys_saved_LIBS
 
-  AC_MSG_CHECKING(whether EC keys are supported)
-  AC_MSG_RESULT([$enable_tls_eckey])
-  TS_ARG_ENABLE_VAR([use], [tls-eckey])
-  AC_SUBST(use_tls_eckey)
-])
-
-AC_DEFUN([TS_CHECK_CRYPTO_NEXTPROTONEG], [
-  enable_tls_npn=yes
-  _npn_saved_LIBS=$LIBS
-
-  TS_ADDTO(LIBS, [$OPENSSL_LIBS])
-  AC_CHECK_FUNCS(SSL_CTX_set_next_protos_advertised_cb SSL_CTX_set_next_proto_select_cb SSL_select_next_proto SSL_get0_next_proto_negotiated,
-    [], [enable_tls_npn=no]
-  )
-  LIBS=$_npn_saved_LIBS
-
-  AC_MSG_CHECKING(whether to enable Next Protocol Negotiation TLS extension support)
-  AC_MSG_RESULT([$enable_tls_npn])
-  TS_ARG_ENABLE_VAR([use], [tls-npn])
-  AC_SUBST(use_tls_npn)
+dnl
+dnl Check OpenSSL Version
+dnl
+AC_DEFUN([TS_CHECK_CRYPTO_VERSION], [
```

Comment:
@zwoop The debian-github job failed with this check. What version is used?

```
checking OpenSSL version... configure: error: in `/var/jenkins/workspace/debian-github/src':
configure: error: requires an OpenSSL version 1.0.2 or greater
```
https://ci.trafficserver.apache.org/job/debian-github/4524/console

Reply:
Pulled docker image and it looks 1.0.1t. Let's upgrade openssl.
```
root@d3a172ce9f2c:/# openssl version
OpenSSL 1.0.1t  3 May 2016
root@d3a172ce9f2c:/# which openssl
/usr/bin/openssl
```

Reply:
Or change the GitHub build to use Debian 9 if there is not a package for Debian 8.